contrib/ovirt/lib/ovirtlago/__init__.py

```diff
@@ -524,7 +524,7 @@ def collect_artifacts(self, output_dir):
         os.makedirs(output_dir)
 
         def _collect_artifacts(vm):
-            with LogTask('%s' % vm.name()):
+            with LogTask('Collecting artifacts from VM %s' % vm.name()):
```

Comment:
For the tasks use base form of the verb instead, gives the same info with less words and makes a little more sense talking about tasks:

  Collect artifacts from VM %s